# Diff: https://project-skyscraper.com
# File: html/index.html
# Timestamp: 2026-05-28 14:24:30 UTC
# Lines: 526 old -> 526 new
# Beautified: yes
--- old/html/index.html
+++ new/html/index.html
@@ -95,7 +95,7 @@
 /*# sourceURL=global-styles-inline-css */
 </style>
 <style id="wp-block-custom-css-inline-css">
-:root :where(.wp-custom-css-a47fe245){position: relative;
+:root :where(.wp-custom-css-e3f8f590){position: relative;
 width: 100%;
 height: 100vh;           /* Plein écran en hauteur */
 max-width: none !important;
@@ -193,7 +193,7 @@
 <body class="home wp-singular page-template page-template-connecting-dots page page-id-551 wp-custom-logo wp-embed-responsive wp-theme-perenne is-block-theme">
 
 <div class="wp-site-blocks">
-<div class="wp-block-group has-custom-css is-layout-flow wp-block-group-is-layout-flow wp-custom-css-a47fe245">
+<div class="wp-block-group has-custom-css is-layout-flow wp-block-group-is-layout-flow wp-custom-css-e3f8f590">
 <p class="wp-block-paragraph"></p>
 
 
@@ -206,7 +206,7 @@
 <!--  -->
 <div style="position: fixed; top: 50%; left: 50%; transform: translate(-50%, -50%); z-index: 10; text-align: center; pointer-events: auto; width: 100%; padding: 20px;">
   <div id="live-visitors">
-    <span class="online-iterations "><strong>5</strong> Live Connections</span>
+    <span class="online-iterations "><strong>3</strong> Live Connections</span>
   </div>
 </div>
 
@@ -519,8 +519,8 @@
 </body>
 </html>
 <!--
-	generated 112 seconds ago
-	generated in 0.305 seconds
-	served from batcache in 0.003 seconds
-	expires in 188 seconds
+	generated 93 seconds ago
+	generated in 0.327 seconds
+	served from batcache in 0.002 seconds
+	expires in 207 seconds
 -->
